extension/scripts/features/company-addiction/ttCompanyAddiction.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+
+(async () => {
+	const devices = await checkDevice();
+	if (devices.mobile || devices.tablet) return "Not supported on mobiles or tablets!";
+
+	featureManager.registerFeature(
+		"Company Addicition Level",
+		"sidebar",
+		() => settings.pages.sidebar.companyAddictionLevel,
+		null,
+		showCompanyAddictionLevel,
+		removeCompanyAddictionLevel,
+		{
+			storage: ["settings.pages.sidebar.companyAddictionLevel", "userdata.job.company_id"],
+		},
+		() => {
+			if (!hasAPIData() || !userdata.job.company_id) return "No API access.";
+		}
+	);
+
+	async function showCompanyAddictionLevel() {
+		await requireSidebar();
+
+		removeCompanyAddictionLevel();
+		addInformationSection();
+		showInformationSection();
+
+		let addiction = await getCompanyAddiction();
+
+		let companyAddictionElement = document.newElement({ type: "span", dataset: { addiction } })
+
+		companyAddictionElement.textContent = addiction.toString();
+
+		document.find(".tt-sidebar-information").appendChild(
+			document.newElement({
+				type: "section",
+				id: "companyAddictionLevel",
+				children: [document.newElement({ type: "a", class: "title", text: "Company Addiction: ", href: LINKS.companyEmployees }), companyAddictionElement],
+			})
+		);
+	}
+
+    async function getCompanyAddiction() {
+        if (ttCache.hasValue("company", "addiction")) {
+			return ttCache.get("company", "addiction");
+		} else {
+            let id = userdata.player_id;
+		    let company_id = userdata.job.company_id;
+
+			const company_employees = (
+				await fetchData("torn", {
+					section: "company",
+					id: company_id,
+					selections: ["employees"],
+					silent: true,
+					succeedOnError: true,
+				})
+			).company_employees;
```

Comment:
You should check for potential error returns. If there is an error it can abort entirely, or throw an error (so it will show up in the feature manager).